defcon/console.lua

```diff
@@ -187,7 +187,13 @@ function M.start(port)
 
 	M.register_command("commands", "Show all commands", function()
 		local s = ""
-		for command,command_data in pairs(commands) do
+		local names = {}
+		for command, command_data in pairs(commands) do
+			table.insert(names, command)
+		end
+		table.sort(names)
+
+		for _,command in pairs(names) do
```

Comment:
Shouldn't it be ipairs()? Is it guaranteed that `pairs()` will iterate in order in a table with keys 1..#names ?

Reply:
True! It's not guaranteed, I haven't noticed it not being sorted, but to be fair it's very likely that I have just missed it. Will fix it.